src/pages/LogIn/index.tsx

```diff
@@ -43,7 +44,7 @@ function LogIn() {
     <section className="flex h-full">
       <div className="text-center bg-white half-section">
         <Logo />
-        {username ? (
+        {username && !isAuthenticated ? (
```

Comment:
I find it weird that in the login page you have logic for when the user is authenticated. I would have expected that logic to live in a page like `/pages/Home/index.tsx` and in the login you can have a useEffect that depends on `isAuthenticated` and if `isAuthenticated` is true, it redirects the user to the Home page.

Reply:
I added that because that block shows the message `Hi {username}` right after the user create's and account but hasn't logged in yet